during_workshop.md

```diff
@@ -87,55 +87,17 @@ cd AcmeSubProject
 Next, we want to create a function that will be triggered by an HTTP request, so we will use the 'HTTP trigger' template.
 
 ```
-func new --name HttpEndpoint --template "HTTP trigger" --authlevel "anonymous"

```

Comment:
This flag didn't work for me locally

Reply:
I see the same; looks like [a known issue](https://github.com/Azure/azure-functions-core-tools/issues/3533)